### PR TITLE
modify master files for 4.13 upgrade docs

### PR DESCRIPTION
**Pull request by PaulAngus**

### CODE REVIEW

**Comment by PaulAngus** [comment]
requesting docbuild

**Comment by PaulAngus** [comment]
I just need a couple of eyes-on this v. basic change to start 4.13 docs

**Comment by rhtyd** [comment]
The only issue is that until now, we don't have a new systemvmtemplate for 4.13.0.0 @PaulAngus rest LGTM.

File: source/_global.rst [hunk 1]
```diff
@@ -24,18 +24,19 @@
 .. |documentation_home| replace:: http://docs.cloudstack.apache.org/
 
 .. Latest version systemvm template name
-.. |sysvm64-name-xen|    replace:: systemvm-xenserver-4.11.3
-.. |sysvm64-name-kvm|    replace:: systemvm-kvm-4.11.3
-.. |sysvm64-name-vmware| replace:: systemvm-vmware-4.11.3
-.. |sysvm64-name-hyperv| replace:: systemvm-hyperv-4.11.3
-.. |sysvm64-name-ovm|    replace:: systemvm-ovm-4.11.3
+
+.. |sysvm64-name-xen|    replace:: systemvm-xenserver-4.13.0
```
Comment (rhtyd) [inline source/_global.rst]: We don't have new 4.13.0 systemvmtemplate. The 4.11.3.0 systemvmtemplate is actually good enough.

Comment (PaulAngus) [reply source/_global.rst]: @rhtyd I *thought* that the changes that you merged for the system vm necessitate a new template?

Comment (rhtyd) [reply source/_global.rst]: @PaulAngus a systemvmtemplate is only necessary when changes are done in cloud-early-config script, in master (https://github.com/apache/cloudstack/pull/3126) all the changes from were in other files which would be patched during startup by systemvm.iso and therefore an explicit/new systemvmtemplate is not necessary.

Comment (PaulAngus) [reply source/_global.rst]: there should be changes to the services which were already started/enabled during OS boot up @rhtyd

Comment (rhtyd) [reply source/_global.rst]: @PaulAngus Yes, this is taken care by bootstrap.sh: https://github.com/apache/cloudstack/blob/master/systemvm/debian/opt/cloud/bin/setup/bootstrap.sh#L67
During patching, first the scripts are patched including bootstrap.sh, which is then executed and will manage/launch services on startup.

Comment (rhtyd) [reply source/_global.rst]: Let me re-confirm and test if we can do without a new systemvmtemplate.

Comment (PaulAngus) [reply source/_global.rst]: I've changed the system vm templates to 4.11.3 for the time being to get the overall changes in.

**Pull request closed without merging**